components/BookingListPreview.js

```diff
@@ -38,13 +48,34 @@ const BookingListPreview = ({ booking }) => {
         </p>
       )}
       <div className="my-2">
-        <Link passHref href={`/bookings/${bookingId}/cancel`}>
-          <a>
-            <button disabled={!isBookingCancelable} className="btn disabled:text-gray-400 disabled:border-gray-400 disabled:cursor-not-allowed">
-              {__('booking_cancel_button')}
-            </button>
-          </a>
-        </Link>
+        { user && user._id === booking.get('createdBy') && (
```

Comment:
lets maybe create a variable for `booking.get('createdBy')` - its used 3 times in the file

Reply:
I personally like better to use the objects directly rather than storing in a variable. Doesn't make the code more understandable or shorter or more efficient.